.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["jason"]
+}
```

Comment:
probably shouldn't re-commit this i imagine scott removed it cause it was messing with things in trackvia-ui